### ISSUE TITLE
Screenreader improvements

### ISSUE THREAD

**Issue by livvyfox** [brickfield/moodle-mod_board#18]
You might already be aware of these issues and logging in case they are useful or you have any advice on workarounds for screenreader users.
Tested with NVDA, Narrator & VoiceOver on Moodle 3.9 using Boost.

1. Count of ratings: No screenreader we tested read a count of the number of stars received on a post.
2. Provide rating: Using NVDA the rating confirmation dialog box opened. NVDA does not read the confirmation screen. You can tab to the OK and Cancel buttons which are read out but not selectable using keyboard while NVDA is running.
3. Delete post: Using NVDA the delete confirmation dialog box opened. NVDA does not read out the the confirmation screen. You can tab to the Delete and Cancel buttons which are read out but not selectable using keyboard while NVDA is running. 
3. Move post between columns: Post should be selectable with a change in focus indicator and confirmation that item has been selected, then selectable space in other column should be selectable in order to drop it too. Does not work on any screenreaders tested
4. Images: No option to add alt text for images added to posts and image title is not read using screenreaders tested.

**Comment by learningtechnologyservices** [comment]
(not karen responding)

(will respond on the rest of points soon)

The "Title" of the image is used for the alt text as below - and was read by NVDA screen reader when tested 

It produces the html <img src="/pluginfile.php/1193/mod_board/images/78/photo_2021-11-09_09-31-58.jpg" alt="Sun rising over Dublin beach" class="mod_board_preview_element">

We are improving the post-it layout, more which should solve most of the issues highlighted.
